Sofa/framework/LinearAlgebra/src/sofa/linearalgebra/CompressedRowSparseMatrixGeneric.h

```diff
@@ -254,7 +254,14 @@ public :
 
     const VecIndex& getRowIndex() const { return rowIndex; }
     const VecIndex& getRowBegin() const { return rowBegin; }
-    Range getRowRange(Index id) const { return Range(rowBegin[id], rowBegin[id+1]); }
+
+    Range getRowRange(Index id) const
+    {
+        if ((unsigned) id+1 >= rowBegin.size()) return Range();
```

Comment:
```suggestion
        if ((unsigned) id >= rowBegin.size()) return Range();
```
`id == rowBegin.size()` would also lead to an out-of-bounds.